modules/ROOT/pages/_partials/include-nav-steps-telemetry-permissions.adoc

```diff
@@ -0,0 +1,17 @@
+// tag::connectionPermission[]
+. Log in to Anypoint Platform using an account that has the Telemetry Exporter Administrator permissions.
+. In the navigation bar or the main Anypoint Platform screen, click *Monitoring*.
```

Comment:
If this is the navigation menu, we are calling this a "sidebar" now.

Reply:
Just "the sidebar" or does it have a name?